lib/grape-swagger/endpoint.rb

```diff
@@ -192,14 +190,17 @@ def response_object(route, markdown)
 
         @definitions[response_model][:description] = description_object(route, markdown)
         # TODO: proof that the definition exist, if model isn't specified
-        memo[value[:code]][:schema] = if route.options[:is_array]
-                                        { 'type' => 'array', 'items' => { '$ref' => "#/definitions/#{response_model}" } }
-                                      else
-                                        { '$ref' => "#/definitions/#{response_model}" }
-                                      end
+        memo[value[:code]][:schema] = route.options[:is_array] ? { 'type' => 'array', 'items' => { '$ref' => "#/definitions/#{response_model}" } } : { '$ref' => "#/definitions/#{response_model}" }
```

Comment:
is this a rubocop recommendation? … it is really a very long line … can you change it back?